ue5osc/__init__.py

```diff
@@ -53,13 +61,13 @@ def set_location(self, x: float, y: float, z: float) -> None:
         self.client.send_message("/set/location", [x, y, z])
 
     def get_rotation(self) -> tuple[float, float, float]:
-        """Returns pitch, yaw, and roll."""
+        """Returns roll, pitch, and yaw."""
         return self.send_and_await("/get/rotation")
 
     def set_yaw(self, yaw: float) -> None:
         """Set the camera yaw in degrees."""
         ue_roll, ue_pitch, _ = self.get_rotation()
-        self.client.send_message("/set/rotation", [ue_pitch, ue_roll, yaw])
+        self.client.send_message("/set/rotation", [ue_roll, ue_pitch, yaw])

```

Comment:
Is this in the local coordinate frame, or the global? That will depend on how it is handled on the Unreal side and we should document it here.

Reply:
This is in the global coordinate frame.

Reply:
Will this work the way we want it to then? I think we want to set `yaw` with respect to the current rotation.